src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -47,6 +48,8 @@
     public static final String MESSAGE_DELETED_TAG_SUCCESS = "Deleted Tag: %1$s";
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_TAG_NOT_DELETED = "There are no tags to delete.";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT_INVALID_INDEX = String.format(
+            MESSAGE_INVALID_COMMAND_FORMAT, "A valid Index was not entered. \n%1$s");
```

Comment:
same as above, is "the index entered is invalid" better?
also, there's no need to capitalise I in index cause user don't know it's a class